phoenix-core/src/main/java/org/apache/phoenix/monitoring/MetricType.java

```diff
@@ -150,6 +150,7 @@ public enum MetricType {
     CLIENT_METADATA_CACHE_MISS_COUNTER("cmcm", "Number of cache misses for the CQSI cache.", LogLevel.DEBUG, PLong.INSTANCE),
     CLIENT_METADATA_CACHE_HIT_COUNTER("cmch", "Number of cache hits for the CQSI cache.", LogLevel.DEBUG, PLong.INSTANCE),
     PAGED_ROWS_COUNTER("prc", "Number of dummy rows returned to client due to paging.", LogLevel.DEBUG, PLong.INSTANCE),
+    STALE_METADATA_CACHE_EXCEPTION_COUNTER("smcem", "Number of StaleMetadataCacheException encountered.", LogLevel.DEBUG, PLong.INSTANCE),
```

Comment:
Why `smcem` ?  What does the last m indicate? (metric?)  If yes, then we don't have in other metrics.

Reply:
I think I wanted to add a `c` for counter and added `m` instead. I will remove it.